src/assets/components/_el-select.scss

```diff
@@ -1,14 +1,16 @@
 @import '../_variables.scss';
 
 .el-select {
-  &.secondary {
-    .el-input {
-      input {
-        border-color: lightgray;
-      }
-    }
-  }
   .el-input {
+    input::placeholder {
+      color:$input-text;
+    }
+    .el-select__caret::before{
+      content: '\e605';
```

Comment:
Looks good in general but you can add the following to make sure the arrow stay in the middle.

.el-select__caret.is-reverse {
    padding-top: 8px;
}